src/applications/gi/reducers/calculator.js

```diff
@@ -69,6 +76,91 @@ export default function (state = INITIAL_STATE, action) {
         ...newState
       };
     }
+    case FETCH_BAH_FAILED: {
+      const { beneficiaryZipFetched } = action;
+      const { error } = action.payload;
+
+      // response mismatch - do nothing
+      if (beneficiaryZipFetched !== state.beneficiaryZipFetched) {
+        return state;
```

Comment:
App will just do nothing if the input doesn't match the zip used for the request.